rules/framework.bzl

```diff
@@ -582,7 +582,7 @@ def _merge_root_infoplists(ctx):
     current_apple_platform = transition_support.current_apple_platform(apple_fragment = ctx.fragments.apple, xcode_config = ctx.attr._xcode_config)
     platform_type = str(current_apple_platform.platform.platform_type)
     apple_mac_toolchain_info = ctx.attr._toolchain[AppleMacToolsToolchainInfo]
-    rule_descriptor = rule_support.rule_descriptor(ctx)
+    rule_descriptor = rule_support.rule_descriptor_no_ctx(platform_type, apple_product_type.static_framework)
```

Comment:
Right, in my other PR I could've passed the attribute we need directly like this 🤦, nice one

Reply:
NP @thiagohmcruz it just seemed like a small thing to fix up and make a bit easier